ui/src/layout/Navigation.tsx

```diff
@@ -54,7 +54,7 @@ class Navigation extends Component<
     public render() {
         const {classes, loggedIn, appStore, navOpen, setNavOpen} = this.props;
         const {showRequestNotification} = this.state;
-        const apps = appStore.getItems();
```

Comment:
This probably should be done in the back end, so that the ordering is consistent with the android app.